symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/BdkConfigParser.java

```diff
@@ -35,7 +35,9 @@ public static JsonNode parse(InputStream inputStream) throws BdkConfigException
         }
 
         try {
-            return YAML_MAPPER.readTree(content);
+            JsonNode jsonNode =  YAML_MAPPER.readTree(content);
```

Comment:
We should probably make BdkConfigParser internal right? 
I see it is already does not have public visibility, maybe annotate it with API guardian?